dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -73,8 +70,17 @@ def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions,skipClear = False
             for row in reader:
                 skipInsert = False
                 for field in fieldOptions:
+                    if row[field] is None:
+                        # If field is empty set to an empty string
+                        row[field] = ""
                     # For each field with options present, modify according to those options
                     options = fieldOptions[field]
+                    if "strip_commas" in options:
+                        # Remove commas from numeric fields
+                        row[field] = row[field].replace(",","")
+                        if row[field] == "":
```

Comment:
Not sure about the SF-133 data, but in many cases it's important to distinguish between a 0 value and a NULL value. If an SF-133 dollar amount comes in as empty, do we know the appropriate way to log that?